templates/tfengine/components/cicd/terraform.tfvars

```diff
@@ -69,20 +69,29 @@ envs = [
           {{- if has .triggers "validate"}}
           run_on_push = {{get .triggers.validate "run_on_push" true}}
           run_on_schedule = "{{get .triggers.validate "run_on_schedule" ""}}"
+          {{- else}}
+          run_on_push = true
+          run_on_schedule = ""
           {{- end}}
         }
         plan = {
           skip = {{not (has .triggers "plan")}}
           {{- if has .triggers "plan"}}
           run_on_push = {{get .triggers.plan "run_on_push" true}}
           run_on_schedule = "{{get .triggers.plan "run_on_schedule" ""}}"
+          {{- else}}
+          run_on_push = true
+          run_on_schedule = ""
```

Comment:
why do we need these? Because those vars are mandatory? If so, the default for a bool variable should be set to `false`.

Reply:
Yeah, the whole object is mandatory and avoiding subkeys result in an error. I decided to put `run_on_push` in true because that's the default specified on the schema.

Doesn't really matter since it is not used 'cause it's skipped but I'll update to false

Reply:
Done